### PR TITLE
Removed prob rate in edit menu

### PR DESCRIPTION
**Pull request by Kasmadei**
Part of https://github.com/kbss-cvut/fta-fmea-ui/issues/417

### CODE REVIEW

File: src/components/dialog/faultEvent/FaultEventCreation.tsx [hunk 1]
```diff
@@ -181,18 +189,20 @@ const FaultEventCreation = ({ useFormMethods, isRootEvent, eventValue, isEditedE
...
-                min={0}
-                max={1}
-                step={0.01}
-                error={!!errors.probability}
-                helperText={errors.probability?.message}
-                className={classes.probability}
-                defaultValue=""
-                {...register("probability")}
-              />
+              {!isSimplified && eventTypeWatch === EventType.BASIC && (
```
Comment (blcham) [inline src/components/dialog/faultEvent/FaultEventCreation.tsx:192]: It is not about simplification. Let's discuss it on standup.

File: src/components/dialog/faultEvent/FaultEventCreation.tsx [hunk 2]
```diff
@@ -16,10 +16,18 @@ interface Props {
...
 
 // TODO: remove ts-ignores and migrate to higher version of react-hook-form
-const FaultEventCreation = ({ useFormMethods, isRootEvent, eventValue, isEditedEvent = false, disabled }: Props) => {
+const FaultEventCreation = ({
+  useFormMethods,
+  isRootEvent,
+  eventValue,
+  isEditedEvent = false,
+  disabled,
+  isSimplified, // Hides props that are needed only for creation
```
Comment (LaChope) [inline src/components/dialog/faultEvent/FaultEventCreation.tsx:29]: No need to have the comments here

**Review by LaChope — CHANGES_REQUESTED** [review]

File: src/components/dialog/faultEvent/FaultEventCreation.tsx (hunk 2 above)
Comment (Kasmadei) [reply src/components/dialog/faultEvent/FaultEventCreation.tsx:29]: Ok, i'll remove it

File: src/components/dialog/faultEvent/FaultEventCreation.tsx (hunk 1 above)
Comment (LaChope) [reply src/components/dialog/faultEvent/FaultEventCreation.tsx]: I agree with `isEditMode`

File: src/components/dialog/faultEvent/FaultEventCreation.tsx [hunk 3]
```diff
@@ -181,18 +189,20 @@ const FaultEventCreation = ({ useFormMethods, isRootEvent, eventValue, isEditedE
...
-                min={0}
-                max={1}
-                step={0.01}
-                error={!!errors.probability}
-                helperText={errors.probability?.message}
-                className={classes.probability}
-                defaultValue=""
-                {...register("probability")}
-              />
+              {!isEditMode && eventTypeWatch === EventType.BASIC && (
```
Comment (blcham) [inline src/components/dialog/faultEvent/FaultEventCreation.tsx:192]: I do not understand why this is not related to EventType.EXTERNAL but it is related to EventType.BASIC

**Review by blcham — CHANGES_REQUESTED** [review]
See my comment

**Comment by blcham** [comment]
To clarify node types I created the following diagram:
image
